src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/telemetry/telemetry.component.ts

```diff
@@ -88,9 +88,27 @@ export class TelemetryComponent extends CdForm implements OnInit {
     this.configForm = this.formBuilder.group(controlsConfig);
   }
 
+  private replacer(key: string, value: any) {
```

Comment:
nit: Can you replace the `any` and use the exact datatype here?

Reply:
Fixed!

Reply:
@nizamial09 I think in this very case, `value` can be `any`thing, since this is a serializer that converts any object into a string (JSON) (check the discussion from TS folks: https://github.com/microsoft/TypeScript/issues/18879).

Reply:
You were right @epuertat

Reply:
I'll revert the commit then.

Reply:
Sorry for the confusion here 😅 @ljflores

Reply:
No worries!